### DISCUSSION TITLE
Can we greet with message on every new issue created/opened not just the first one.

### DISCUSSION THREAD

**Discussion by rohitthakur2590** [community/community#26509] [Actions]
<table><tbody>
<tr><td>
<span>name</span>: <span>Greetings</span>
</td></tr>
<tr>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td>
<span>on</span>: issues</td>
</tr>
<tr>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td>
<span>jobs</span>:</td>
</tr>
<tr>
<td> </td>
<td>
<span>greeting</span>:</td>
</tr>
<tr>
<td> </td>
<td>
<span>runs-on</span>: <span>ubuntu-latest</span>
</td>
</tr>
<tr>
<td> </td>
<td>
<span>steps</span>:</td>
</tr>
<tr>
<td> </td>
<td>- <span>uses</span>: <span>actions/first-interaction@v1</span>
</td>
</tr>
<tr>
<td> </td>
<td>
<span>with</span>:</td>
</tr>
<tr>
<td> </td>
<td>
<span>repo-token</span>: <span>${{ secrets.GITHUB_TOKEN }}</span>
</td>
</tr>
<tr>
<td> </td>
<td>
<span>issue-message</span>: <span><span>'</span>@${{ github.actor }} just so you are aware we have a dedicated Working Group for the network.</span>
</td>
</tr>
<tr>
<td> </td>
<td>
<p><span>pr-message</span>: <span><span>'</span>@${{ github.actor }} just so you are aware we have a dedicated Working Group for the network.</span></p>
<p> </p>
<p>This workflow greets only on the very first issue created by the user. I want to display the same message on every new issue created/opened</p>
</td>
</tr>
</tbody></table>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/rohitthakur2590">@rohitthakur2590</a> ,</p>
<p>If you want to add comments when a pull request/issue opened, you could use another action <a href="https://github.com/marketplace/actions/create-or-update-comment" rel="nofollow noopener">Create or Update Comment</a> to implete this . </p>
<p>Please check my example workflow yml: </p>
<pre><code>name: Greetings

on:
  issues:
    types: [opened]
  pull_request:
    types: [opened]
jobs:

  greeting:
    runs-on: ubuntu-latest
    steps:
    - name: Create comment for issue
      if: github.event_name =='issues' uses: peter-evans/create-or-update-comment@v1
      with:
        issue-number: ${{tojson(github.event.issue.number)}}
        body: |
            @${{ github.actor }} just so you are aware we have a dedicated Working Group for the network.
    - name: Create comment for PR
      if: github.event_name == 'pull_request'
      uses: peter-evans/create-or-update-comment@v1
      with:
        issue-number: ${{tojson(github.event.number)}}
        body: |
            @${{ github.actor }} just so you are aware we have a dedicated Working Group for the network.
</code></pre>

**Comment by rohitthakur2590** [comment]
<p><a class="mention" href="/yanjingzhu">@yanjingzhu</a>  That is super cool !! thank you</p>
